### ISSUE TITLE
Archive is deleted on close when using prefix to open it from another archive

### ISSUE THREAD

**Issue by rohitab** [FarGroup/FarManager#683]
### Far Manager version

3.0.6141.0 x64

### OS version

10.0.22624.1755

### Steps to reproduce

1. Open any archive, say `test.zip` using ArcLite
2. From inside the archive, open another archive using `arc:` plugin prefix, e.g. `arc:%TEMP%\sample.zip`
3. The new archive `sample.zip` will open up in the Active panel, as expected
4. Press `Ctrl+PgUp` to close the panel
5. You will be returned back to the original archive `test.zip`, as expected

### Expected behavior

The file `%TEMP%\sample.zip` should _not_ be deleted.

### Actual behavior

The file `%TEMP%\sample.zip` is deleted on closing the panel in Step 4 above.

The file is deleted by the following piece of code. The comment below, translated to English is _"// remove file from previous plugin_".
https://github.com/FarGroup/FarManager/blob/850b342c004ecd97d69e98c0f94a316eae49b38d/far/filelist.cpp#L5667-L5670

Please note that this happens with other plugins as well. For example, if you open a `.dll` in [PE Analyzer](https://sourceforge.net/projects/farplugs/files/PEAnalyzer/) or a `.class` file [JClassInfo](https://sourceforge.net/projects/farplugs/files/JClassInfo/), and then use the `arc:` prefix, the archive is deleted.

This is a recent issue, due to a change I requested in #680 to enable recursive plugin panels for command-line prefix calls. Once again, apologies for not testing it thoroughly before requesting the change to be made.

**Comment by w17** [comment]
6152
new options: [-r [-x[d]]]
**arc:-r archive** to open in recursion mode
-x[d] -- remove hostfile [directory] on close

**Comment by rohitab** [comment]
I tested version `3.0.6153.0 x64`. This issue is now resolved.

@w17, the `-xf` and `-xd` options are not working, although `-x` by itself works just fine. The issue appears to be due to a typo, i.e. an extra space after `xf` and `xd` in lines `177` and `180` respectively, in the following code:

https://github.com/FarGroup/FarManager/blob/131e71862c2639c7a02240bd9cfc6b16bda93ca0/plugins/arclite/cmdline.cpp#L177-L182

Also, I'm not sure if it is by design, but the `-x` option only works if `-r` is also specified, and only works if it is called from inside another archive.

I'm closing this ticket since the reported issue is resolved.
